### DISCUSSION TITLE
US-centric field names

### DISCUSSION THREAD

**Discussion by missmatsuko** [craftcms/commerce#2324] [Ideas]
### Description
There's a couple naming choices in Commerce that are very specific to the US. I'm not sure if this is standard practice for eCommerce systems, but I find it a bit confusing, especially when we're serving countries other than the US.

e.g.
1. [zipCode](https://craftcommerce.com/docs/address-model#zipcode) - A ZIP Code is a postal code used by the United States Postal Service (USPS) [[Wikipedia]](https://en.wikipedia.org/wiki/ZIP_Code). I think a more generic term would be "Post Code", "Postal Code", or perhaps "Address Code". Calling this field `zipCode` makes me think that there could be ZIP validation on it.
2. [stateName, stateText](https://craftcommerce.com/docs/address-model#statetext) - A lot of countries don't use states, and may use provinces or districts or other region names instead. Even US addresses may have States, Districts, Territories, and maybe other non-State "States". I think a generic term that's often used is "Administrative Area".

### Steps to reproduce

1.
2.

### Additional info

- Craft version:
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by lukeholder** [comment]
Thanks for the feedback. Will look to improve this.

**Comment by lukeholder** [comment]
Just realized that the labels for address fields should already be translated.

https://github.com/craftcms/commerce/blob/develop/src/models/Address.php#L196-L217

You can you static translations (and pass in the `commerce` category). Info here: https://docs.craftcms.com/v3/static-translations.html#prep-your-messages

**Comment by missmatsuko** [comment]
It's not really just the labels/translations. Naming the field `zipCode` is confusing because it seems like it's expecting an actual valid ZIP code, rather than any type of postal code. 

It makes sense to make the labels generic, unless it's for a store that serves only one country.

**Comment by nfourtythree** [comment]
Hi @missmatsuko 

This would be a large breaking change for something like this. We will need to look at this in a major version release.
